system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -235,7 +303,9 @@ class TopSitesEdit extends React.Component {
                   key={link.guid || link.url}
                   dispatch={this.props.dispatch}
                   link={link}
-                  index={index} />)}
+                  index={index}
+                  intl={this.props.intl}
+                  editMode={true} />)}
```

Comment:
Yeah, I've thought about it that little too but wasn't sure how to do it easily with the way the UI is laid out with the overlay on top of the entire page then the modal on top. But I'm sure there is a way to do it, or at least simplify it a bit if we dedicate some brainpower to it 👍